### PR TITLE
Fixed bug in House_Prices_Regression_Interpret tutorial

### PR DESCRIPTION
**Pull request by dbish**
Described in issue: [1012](https://github.com/pytorch/captum/issues/1012)

**Background**
There's a line that must have been edited/added at some point that assumes more then one tensor is being returned from lc.attribute, but there's only one (since only one tensor is passed in):

lc_attr_test = lc.attribute(X_test, n_steps=100, attribute_to_layer_input=True)

# shape: test_examples x size_hidden
lc_attr_test = lc_attr_test[0]
The second line here of setting lc_attr_test to the 0th index, then sets it to the tensor index instead of the first tensor, which in turn in the next cell means that "lc_attr_test.shape[1]" throws an exception. 

**Changes**
Fix is to just take out the reassigning of "lc_attr_test = lc_attr_test[0]".

**Testing**
Tested running the notebook before the change (which showed the error described) and after, which produced the plot which was originally shown in the static tutorial page and matches expectations now (see attachment)
Screen Shot 2022-08-18 at 9 59 25 PM
.

### CODE REVIEW

**Comment by dbish** [comment]
Looks like this tutorial hasn't been updated in a bit, it works now, but there are a few new warnings that didn't show up before in the notebook output without changing any additional code besides the line mentioned. Big one is a function deprecation warning from sklearn https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html. Maybe worth making another update or opening a new task to clean those up.

File: tutorials/House_Prices_Regression_Interpret.ipynb [hunk 1]
```diff
@@ -63,7 +63,48 @@
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (vivekmig) [inline tutorials/House_Prices_Regression_Interpret.ipynb:66]: nit: Can we revert the change in these lines and 474-487? This will avoid including your local machine path in the notebook. We will address the deprecation warnings separately.

**Review by vivekmig — APPROVED** [review]
Looks great, thanks so much for fixing this! Just one small nit.

File: tutorials/House_Prices_Regression_Interpret.ipynb (hunk 1 above)
Comment (dbish) [reply tutorials/House_Prices_Regression_Interpret.ipynb:66]: makes sense. i'll clean those lines up
